File: CPP/ManagerCustomer.cpp
#include <bits/stdc++.h>
#include "Manager.cpp"
#include "CustomerService.cpp"
#pragma once
using namespace std;

class ManagerCustomer : public CustomerService, public Manager {
private:
    string rencanaPeningkatanPelayanan;
    string targetPelayanan;

public:
    ManagerCustomer(string nama = "", string nohp = "", string idPegawai = "", int LamaTahunBekerja = 0, 
                    int JumlahKeluhan = 0, float RatingPelanggan = 0, int JumlahAnggotaTim = 0, int TargetKinerja = 0, 
                    string rencanaPeningkatanPelayanan = "", string targetPelayanan = "") 
        : CustomerService(nama, nohp, idPegawai, LamaTahunBekerja, JumlahKeluhan, RatingPelanggan), 
          Manager(nama, nohp, idPegawai, LamaTahunBekerja, JumlahAnggotaTim, TargetKinerja) {
        this->rencanaPeningkatanPelayanan = rencanaPeningkatanPelayanan;
        this->targetPelayanan = targetPelayanan;
    }

    string getRencanaPeningkatanPelayanan() {
        return rencanaPeningkatanPelayanan;
    }

    void setRencanaPeningkatanPelayanan(string rencanaPeningkatanPelayanan) {
        this->rencanaPeningkatanPelayanan = rencanaPeningkatanPelayanan;
    }

    string getTargetPelayanan() {
        return targetPelayanan;
    }

    void setTargetPelayanan(string targetPelayanan) {
        this->targetPelayanan = targetPelayanan;
    }
};

File: Python/ManagerCustomer.py
from Manager import Manager
from CustomerService import CustomerService

class ManagerCustomer(CustomerService, Manager):
    def __init__(self, nama="", nohp="", idPegawai="", LamaTahunBekerja=0, JumlahKeluhan=0, RatingPelanggan=0, 
                 JumlahAnggotaTim=0, TargetKinerja=0, rencanaPeningkatanPelayanan="", targetPelayanan=""):
        
        CustomerService.__init__(self, nama, nohp, idPegawai, LamaTahunBekerja, JumlahKeluhan, RatingPelanggan)
        Manager.__init__(self, nama, nohp, idPegawai, LamaTahunBekerja, JumlahAnggotaTim, TargetKinerja)

        self.__rencanaPeningkatanPelayanan = rencanaPeningkatanPelayanan
        self.__targetPelayanan = targetPelayanan

    def getRencanaPeningkatanPelayanan(self):
        return self.__rencanaPeningkatanPelayanan
    
    def setRencanaPeningkatanPelayanan(self, rencanaPeningkatanPelayanan):
        self.__rencanaPeningkatanPelayanan = rencanaPeningkatanPelayanan

    def getTargetPelayanan(self):
        return self.__targetPelayanan
    
    def setTargetPelayanan(self, targetPelayanan):
        self.__targetPelayanan = targetPelayanan
